Libraries/Opc.Ua.Security.Certificates/Common/Oids.cs

```diff
@@ -0,0 +1,157 @@
+/* ========================================================================
+ * Copyright (c) 2005-2020 The OPC Foundation, Inc. All rights reserved.
+ *
+ * OPC Foundation MIT License 1.00
+ * 
+ * Permission is hereby granted, free of charge, to any person
+ * obtaining a copy of this software and associated documentation
+ * files (the "Software"), to deal in the Software without
+ * restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following
+ * conditions:
+ * 
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ * OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * The complete license agreement can be found here:
+ * http://opcfoundation.org/License/MIT/1.00/
+ * ======================================================================*/
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+using System;
+using System.Security.Cryptography;
+
+namespace Opc.Ua.Security.Certificates
+{
+    /// <summary>
+    /// Oid constants defined for ASN encoding/decoding.
```

Comment:
Does Oid mean Open ID ? Could you add a link, were this constants are coming from?

Reply:
Technically it means object identifier, but basically these days it just means oid.  Sort of like how no one knows LASER is Light Amplification by Stimulated Emission of Radiation.

Like any IDs they don't come from one spec, they come from several.  e.g. 1.2.840.10045.4.3 comes from the bottom of page 16 in https://tools.ietf.org/html/rfc5480.